packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -40,178 +39,209 @@ import { API_ERROR_CODES } from 'Constants/api-error-codes';
+    ({ handleBack, handleViewComplete, handleSelectionNext, selected_country }: TIDVDocumentSubmitProps) => {
+        const { client, ui } = useStore();
+        const { account_settings, getChangeableFields } = client;
+        const { is_mobile, is_desktop } = ui;
 
-    const visible_settings = ['first_name', 'last_name', 'date_of_birth'];
-    const side_note_image = <PoiNameDobExample />;
+        const IDV_NOT_APPLICABLE_OPTION = React.useMemo(() => getIDVNotApplicableOption(), []);
+        const visible_settings = ['first_name', 'last_name', 'date_of_birth'];
+        const side_note_image = <PoiNameDobExample />;
 
-    const form_initial_values = filterObjProperties(account_settings, visible_settings) as {
-        [Property in keyof TPersonalDetailsForm]: string;
-    };
+        const form_initial_values = filterObjProperties(account_settings, visible_settings) as {
+            [Property in keyof TPersonalDetailsForm]: string;
+        };
 
-    if (form_initial_values.date_of_birth) {
-        form_initial_values.date_of_birth = toMoment(form_initial_values.date_of_birth).format('YYYY-MM-DD');
-    }
+        if (form_initial_values.date_of_birth) {
+            form_initial_values.date_of_birth = toMoment(form_initial_values.date_of_birth).format('YYYY-MM-DD');
+        }
+
+        const changeable_fields = getChangeableFields();
+
+        const initial_values: TIdvDocumentSubmitForm = {
+            document_type: {
+                id: '',
+                text: '',
+                value: '',
+                example_format: '',
+            },
+            document_number: '',
+            ...form_initial_values,
+            confirmation_checkbox: false,
+        };
 
-    const changeable_fields = getChangeableFields();
-
-    const initial_values: TIdvDocumentSubmitForm = {
-        document_type: {
-            id: '',
-            text: '',
-            value: '',
-            example_format: '',
-        },
-        document_number: '',
-        confirmation_checkbox: false,
-        ...form_initial_values,
-    };
-
-    const validateFields = (values: TIdvDocumentSubmitForm) => {
-        const errors: FormikErrors<Record<keyof TIdvDocumentSubmitForm, string>> = {};
-        const { document_type, document_number, document_additional } = values;
-        const needs_additional_document = !!document_type.additional;
-
-        errors.document_type = isDocumentTypeValid(document_type);
-        if (!shouldHideHelperImage(document_type?.id)) {
-            if (needs_additional_document) {
-                errors.document_additional = isAdditionalDocumentValid(document_type, document_additional);
+        const validateFields = (values: TIdvDocumentSubmitForm) => {
+            const errors: FormikErrors<Record<keyof TIdvDocumentSubmitForm, string>> = {};
+            const { document_type, document_number, document_additional } = values;
+
+            if (document_type.id === IDV_NOT_APPLICABLE_OPTION.id) {
```

Comment:
document_type?.id

Reply:
it is also not necessary, as the field is defined with initial values